test/core/tsi/crl_ssl_transport_security_test.cc

```diff
@@ -54,103 +53,88 @@ const char* kSslTsiTestCrlSupportedCrlDirMissingIntermediate =
 const char* kSslTsiTestCrlSupportedCrlDirMissingRoot =
     "test/core/tsi/test_creds/crl_data/crls_missing_root/";
 const char* kSslTsiTestFaultyCrlsDir = "bad_path/";
+const std::string kRevokedKeyPath =
```

Comment:
The style guide prohibits global variables of types that are not trivially destructible, which includes `std::string`: https://google.github.io/styleguide/cppguide.html#Static_and_Global_Variables